CONTRIBUTING.md

```diff
@@ -87,184 +61,63 @@ A: There is no strict definition however examples of substantive changes are:
-
-`.env`
-```
-BB20_CONTEXT=../../bluebutton-web-server
-CERTSTORE=../../bb.dev/certstore
-BFD_DIR=../
-SYNTHETIC_DATA=./synthetic-data
-```
-
-`docker-compose.bb2.yml`
+#### Opening A PR
 
+To contribute work back to the BFD your branch must be pushed to the `CMSgov/beneficiary-fhir-data` repository.
 ```
-version: '3.3'
-
-services:
-  bbdb:
-    image: postgres
-    environment:
-      - POSTGRES_DB=bluebutton
-      - POSTGRES_PASSWORD=toor
-  bb20:
-    build:
-      context: ${BB20_CONTEXT}
-      dockerfile: Dockerfile
-    command: python3 manage.py runserver 0.0.0.0:8000
-    environment:
-      - DJANGO_SETTINGS_MODULE=hhs_oauth_server.settings.dev
-      - DJANGO_FHIR_CERTSTORE=/certstore
-      - DATABASES_CUSTOM=postgres://postgres:toor@bbdb:5432/bluebutton
-      - OAUTHLIB_INSECURE_TRANSPORT=true
-      - DJANGO_DEFAULT_SAMPLE_FHIR_ID="20140000008325"
-      - DJANGO_SECURE_SESSION=False
-      - DJANGO_MEDICARE_LOGIN_URI=http://127.0.0.1:8080?scope=openid%20profile&client_id=bluebutton
-      - DJANGO_SLS_USERINFO_ENDPOINT=http://msls:8080/userinfo
-      - DJANGO_SLS_TOKEN_ENDPOINT=http://msls:8080/token
-      - FHIR_URL=https://bfd.local:9954/v1/fhir/
-    ports:
-      - "8000:8000"
-    links:
-      - "bfd:bfd.local"
-    volumes:
-      - ${BB20_CONTEXT}:/code
-      - ${CERTSTORE}:/certstore
-    depends_on:
-      - bbdb
-```
-
-The BlueButton 2 system also requires an Identity Provider
-`docker-compose.msls.yml`
+git push origin <your username>/<your feature name>
+# To make sure "origin" points to CMSgov/beneficiary-fhir-data run
+# git remote -v
+# if a different remote points to CMSgov/beneficiary-fhir-data
+# replace "origin" with that remotes name in the origional command
 ```
-version: '3.3'
+In order to obtain permission to do this contact a github administrator or reach out on [#bfd](https://cmsgov.slack.com/archives/C010WDXAZFZ).
+Once pushed, [open a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) and use this [PR template](https://github.com/CMSgov/cms-oeda-dasg) found in .github
 
-services:
+Please fill out each section of the body of the PR or quickly justify why the section does not apply to this particular change.
+Reviewers will automatically be suggested or requested based on the projects CODEOWNERS file, feel free to add other reviewers if desired.
+Once all automated checks pass and two reviewers have approved your pull request, a code owner from the core team will do a final review and make the decision to merge it in or not.
 
-  msls:
-    build:
-      context: ../../bb.dev/msls
-      dockerfile: Dockerfile
-    command: msls
-    ports: 
-      - "8080:8080"
-```
+If you have any questions feel free to reach out on the [#bfd](https://cmsgov.slack.com/archives/C010WDXAZFZ) channel in CMS slack!
 
-Bringing these systems up together:
+**FAQ**
```

Comment:
Isn't this duplicative of the FAQ section above?

Reply:
It looks like the question about a rejected PR (lines 121-123) was moved here - should we move it somewhere else?